main.go

```diff
@@ -10,6 +10,8 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var version = "0.0.0"
```

Comment:
but where in the `go build` command is it being set? for example, there should be an `-X main.version=...` somewhere. did I miss it?